superset-frontend/src/explore/components/controls/DndColumnSelectControl/OptionWrapper.tsx

```diff
@@ -107,6 +121,51 @@ export default function OptionWrapper(
     },
   });
 
+  const shouldShowTooltip =
+    (!isDragging && tooltipTitle && label && tooltipTitle !== label) ||
+    (!isDragging &&
+      labelRef &&
+      labelRef.current &&
+      labelRef.current.scrollWidth > labelRef.current.clientWidth);
```

Comment:
```suggestion
      labelRef?.current?.scrollWidth > labelRef.current.clientWidth);
```
If you're feelin' fancy ¯\\\_(ツ)_/¯